test/templates/message-strip/index.njk

```diff
@@ -0,0 +1,336 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "./component.njk" import messageStrip %}
+{% from "./../overlay.njk" import overlay, overlay_toggle %}
+
+
+{% block content %}
+
+<style>
+    body {
+        margin: 0;
+        padding: 20px;
+        font-size: initial;
+        line-height: initial;
+    }
+</style>
+
+<h1>Message Strip</h1>
+
+<!-- output the component example and the code snippet -->
+{% set example %}
```

Comment:
I know this is not part of this ticket, but can we optimize the test page and add loops instead of copying the same code for each message strip?
This will reduce the code in the test file